src/features/user/BulkCodes/index.tsx

```diff
@@ -51,7 +53,7 @@ export default function BulkCodes({
           {
             _scope: 'map',
             currency: planetCashAccount.currency,
-            tenant: TENANT_ID,
+            tenant: tenantID || DEFAULT_TENANT,
```

Comment:
You could use the `getTenantID()` util function while assigning a value to `tenant`. Otherwise, you'll repeat the code, and also not consider `process.env.TENANTID`